libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -213,17 +226,26 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+        uint8_t reenable_use;
+        uint8_t airspeed_failure_basic_failed_count=0;
+        uint8_t airspeed_failure_basic_success_count=0;
+        bool    airspeed_failure_basic_renable=false;
+        uint32_t airspeed_failure_last_check=0;
+        uint32_t airspeed_failure_last_reset=0;
     } state[AIRSPEED_MAX_SENSORS];
 
     // current primary sensor
     uint8_t primary;
     
+
     void read(uint8_t i);
     // return the differential pressure in Pascal for the last airspeed reading for the requested instance
     // returns 0 if the sensor is not enabled
     float get_pressure(uint8_t i);
     void update_calibration(uint8_t i, float raw_pressure);
     void update_calibration(uint8_t i, const Vector3f &vground, int16_t max_airspeed_allowed_during_cal);
+    bool check_airspeed_health_basic(uint8_t i);
```

Comment:
I'm not sure I like the word "basic" throughout all of this. Either way, rename to check_health_basic. You don't need to name things airspeed in the airspeed library

Reply:
Sure no probs